Makefile

```diff
@@ -249,6 +249,12 @@ pao-functests-update-only: $(BINDATA)
 	hack/show-cluster-version.sh
 	hack/run-test.sh -t "test/e2e/performanceprofile/functests/0_config test/e2e/performanceprofile/functests/2_performance_update test/e2e/performanceprofile/functests/3_performance_status test/e2e/performanceprofile/functests/7_performance_kubelet_node test/e2e/performanceprofile/functests/9_reboot" -p "-v -r --fail-fast --flake-attempts=2 --timeout=5h --junit-report=report.xml" -m "Running Functional Tests"
 
+.PHONY: pao-functests-update-only-hypershift
+pao-functests-update-only-hypershift: $(BINDATA)
+	@echo "Cluster Version"
+	hack/show-cluster-version.sh
+	# hack/run-test.sh -t "test/e2e/performanceprofile/functests/0_config test/e2e/performanceprofile/functests/2_performance_update test/e2e/performanceprofile/functests/3_performance_status test/e2e/performanceprofile/functests/7_performance_kubelet_node test/e2e/performanceprofile/functests/9_reboot" -p "-v --label-filter="!openshift" -r --fail-fast --flake-attempts=2 --timeout=5h --junit-report=report.xml" -m "Running Functional Tests"
```

Comment:
why is this commented out ?

Reply:
Since these test suites are not fully supported yet. The intention of this PR is to integrate a hypershift lane to the CI (that will trigger these tests once supported).

Reply:
So why not wait for the support and then merge this without the comment ?  (we can keep the PR open of course)

Reply:
@yanirq We want this make target to be used as a placeholder so that when we set up a new lane on the release repo we'll invoke this make target. 

@rbaturov Let's remove the commented line. I also suggest to already create a PR of openshift/release and refer to it here to make the goal more clear.
Similar to what was done here: https://github.com/openshift/cluster-node-tuning-operator/pull/995

Reply:
Done @yanirq @Tal-or